app/package.json

```diff
@@ -41,23 +41,28 @@
   },
   "dependencies": {
     "api-problem": "^6.1.4",
+    "aws-sdk": "^2.785.0",
     "axios": "^0.21.0",
     "axios-oauth-client": "^1.3.0",
     "axios-token-interceptor": "^0.2.0",
     "body-parser": "^1.19.0",
+    "bytes": "^3.1.0",
     "compression": "^1.7.4",
     "config": "^3.3.2",
     "crypto": "^1.0.1",
     "express": "^4.17.1",
     "express-rate-limit": "^5.1.3",
+    "express-rate-limiter": "^1.3.1",
```

Comment:
Why do we now have two express-rate-limit packages? Should only have one of them if possible and drop the other.

Reply:
yup, we don't use that one... removing express-rate-limiter